extensions/positron-python/src/test/positron/testElectron.ts

```diff
@@ -240,22 +240,11 @@ export async function downloadAndUnzipPositron(): Promise<{ version: string; exe
         case 'darwin':
             suffix = '.dmg';
             break;
-        case 'linux':
-            suffix = '.deb';
-            break;
-        case 'win32':
-            suffix = '.exe';
-            break;
```

Comment:
Removed this as it doesn't seem like we support linux/win32 anyways? See lower down in this function where we look for the `executablePath` -- only mac is supported there.